driver/makedepend.ml

```diff
@@ -653,17 +653,16 @@ let run_main argv =
          "<file> Read additional NUL separated command line arguments from \n\
          \      <file>"
   ];
-  let usage =
-    Printf.sprintf "Usage: %s [options] <source files>\nOptions are:"
-                   (Filename.basename Sys.argv.(0))
-  in
-  Clflags.parse_arguments argv (add_dep_arg (fun f -> Src (f, None))) usage;
+  let program = Filename.basename Sys.argv.(0) in
+  Compenv.parse_arguments (ref argv)
+      (add_dep_arg (fun f -> Src (f, None))) program;
```

Comment:
It looks like this change is making `ocamldep -help` fail with:
```
Fatal error: exception Compenv.Exit_with_status(0)
```
which in turns make `ocamlfind ocamldep` unconditionally fail

Reply:
xref #9798